my-app/src/components/header/header.css

```diff
@@ -0,0 +1,127 @@
+.app .header {
+    display: flex;
```

Comment:
[global]: take your style order with priority. such as "position", "display", "margin", ... in the top of css selector.